Improve descriptions tag (#2772)
near/docs: docs/smart-contracts/anatomy/collections.md

---
id: collections
title: Collections

> description: "Learn NEAR's native and SDK collections to efficiently store, access, and manage smart contract data."

Reviewer: ```suggestion
description: "Efficiently store, access, and manage data in smart contracts."
```
